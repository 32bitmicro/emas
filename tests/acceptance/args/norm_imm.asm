lw r0, 0
lw r1, 1
lw r2, -1
lw r3, -32768
lw r4, 32767
lw r5, 65535
lw r6, 65535
lw r7, 65535
