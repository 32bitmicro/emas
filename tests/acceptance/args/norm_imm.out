@ 0x0000 : 0x4000  /  010 000 0 000 000 000  /  16384
@ 0x0001 : 0x0000  /  000 000 0 000 000 000  /  0
@ 0x0002 : 0x4040  /  010 000 0 001 000 000  /  16448
@ 0x0003 : 0x0001  /  000 000 0 000 000 001  /  1
@ 0x0004 : 0x4080  /  010 000 0 010 000 000  /  16512
@ 0x0005 : 0xffff  /  111 111 1 111 111 111  /  -1
@ 0x0006 : 0x40c0  /  010 000 0 011 000 000  /  16576
@ 0x0007 : 0x8000  /  100 000 0 000 000 000  /  -32768
@ 0x0008 : 0x4100  /  010 000 0 100 000 000  /  16640
@ 0x0009 : 0x7fff  /  011 111 1 111 111 111  /  32767
@ 0x000a : 0x4140  /  010 000 0 101 000 000  /  16704
@ 0x000b : 0xffff  /  111 111 1 111 111 111  /  65535
@ 0x000c : 0x4180  /  010 000 0 110 000 000  /  16768
@ 0x000d : 0xffff  /  111 111 1 111 111 111  /  65535
@ 0x000e : 0x41c0  /  010 000 0 111 000 000  /  16832
@ 0x000f : 0xffff  /  111 111 1 111 111 111  /  65535
